code/FulpstationCode/glowsticks.dm

```diff
@@ -0,0 +1,59 @@
+/obj/item/flashlight/glowstick/military
```

Comment:
These lads used anywhere else in the code, or are they just sorta their own thing?

Reply:
The military glowsticks appear inside SecVend machines. If they don't have a color specified after their type it's a green glowstick.

Reply:
They spawning on brigdocs and/or EMTs, or is it a seperate feature?

NBD if its both imo tho.

Reply:
It's a separate feature. Security and Engineering have access to pouches which can hold some glowsticks for marking and or lighting up areas.